tests/README.md

```diff
@@ -9,9 +9,9 @@
 ### All tests
 
 ```bash
-go test -v setup_test.go        # Only needs to be run once.
+go test -v assets/setup_test.go        # Only needs to be run once.
 go test -v ./scalers_go/...
-go test -v cleanup_test.go      # Skip if you want to keep testing.
+go test -v assets/cleanup_test.go      # Skip if you want to keep testing.
```

Comment:
Personally I don't see those as assets and would keep them in the root folder as well given running tests is there as well.

Reply:
The reason to do it is for skipping them from the  lookup easier, instead of skipping them one by one, we can skip the whole folder

Reply:
Personally I'd name this folder `helpers` or `util`

Reply:
I can live with util